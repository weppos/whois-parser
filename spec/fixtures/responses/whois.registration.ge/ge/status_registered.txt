   Domain Name: GOOGLE.GE
   Creation Date: 2006-07-28
   Registry Expiry Date: 2022-07-29
   Registrar: proservice ltd
   Domain Status: Ok
   Registrant: Google LLC
   Admin Name: Google LLC
   Admin Email: dns-admin@google.com
   Tech Name: Google LLC
   Tech Email: dns-admin@google.com
   Name Server: ns4.google.com
   Name Server: ns3.google.com
   Name Server: ns2.google.com
   Name Server: ns1.google.com


TERMS OF USE:
The WHOIS service is provided solely for informational purposes.
You are not authorized to access or query our Whois
database through the use of electronic processes that are high-volume and
automated except as reasonably necessary to register domain names or
modify existing registrations; the Data is provided by NIC for
information purposes only, and to assist persons in obtaining information
about or related to a domain name registration record. NIC does not
guarantee its accuracy. By submitting a Whois query, you agree to abide
by the following terms of use: You agree that you may use this Data only
for lawful purposes and that under no circumstances will you use this Data
to: (1) allow, enable, or otherwise support the transmission of mass
unsolicited, commercial advertising or solicitations via e-mail, telephone,
or facsimile; or (2) enable high volume, automated, electronic processes
that apply to NIC (or its computer systems). The compilation,
repackaging, dissemination or other use of this Data is expressly
prohibited without the prior written consent of NIC. You agree not to
use electronic processes that are automated and high-volume to access or
query the Whois database except as reasonably necessary to register
domain names or modify existing registrations. NIC reserves the right
to restrict your access to the Whois database in its sole discretion to ensure
operational stability.  NIC may restrict or terminate your access to the
Whois database for failure to abide by these terms of use. NIC
reserves the right to modify these terms at any time.