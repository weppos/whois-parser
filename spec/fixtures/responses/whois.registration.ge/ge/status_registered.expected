#domain
  %s == "google.ge"

#domain_id
  %s %ERROR{AttributeNotSupported}

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2006-07-28}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-07-29}

#registrar
  %s %CLASS{registrar}
  %s.id           == nil
  %s.name         == "proservice ltd"
  %s.organization == nil
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Google LLC"
  %s[0].organization  == nil
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Google LLC"
  %s[0].organization  == nil
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == "dns-admin@google.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Google LLC"
  %s[0].organization  == nil
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == "dns-admin@google.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"